make/run-cicd-script.mk

```diff
@@ -1,7 +1,7 @@
 
-GH_OWNER=codeready-toolchain
+GH_OWNER=mfrancisc
```

Comment:
is it intended?

Reply:
yes, this PR depends on the changes from: https://github.com/codeready-toolchain/toolchain-cicd/pull/120

Reply:
once verified that tests are passing, I can merge the cicd changes and then update this PR and merge it.

Reply:
makes sense, I usually add reminders but I forgot to do that here. I've clarified that in the description hopefully. Sorry for the confusion and thanks for the review.

Reply:
Thanks for the added description @mfrancisc 
and thanks for the suggestion @xcoulon! I had the same question, and although I am looking after the description change, I found the file changes first and this conversation was helpful!